### PR TITLE
 LLVM: Update to 11.1

### PR DESCRIPTION
**Pull request by abpostelnicu**
ALso update include-what-you-use base revision since it's dependent on LLVM

- [x ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by Philhoy54 — APPROVED** [review]

**Review by Philhoy54 — APPROVED** [review]

File: Formula/llvm.rb [hunk 1]
```diff
@@ -36,13 +36,6 @@ class Llvm < Formula
...
-
-    # Regression in LLDB by D89156 on MacOS, fixed by D95683.
-    # https://reviews.llvm.org/D95683
-    patch do
-      url "https://raw.githubusercontent.com/Homebrew/formula-patches/698a42e95e2c982c1d11f1b7d3293ba3aac677ee/llvm/lldb.patch"
-      sha256 "b0dd8153370de8333dc57f950976bc87413c0b053bd30d9b8b10540094923b88"
-    end
   end
 
   livecheck do
```
Comment (Philhoy54) [inline Formula/llvm.rb:48]: - - https://wiki.python.org/moin/LocalUserGroups?action=raw

**Review by Philhoy54 — APPROVED** [review]

File: Formula/llvm.rb (hunk 1 above)
Comment (abpostelnicu) [reply Formula/llvm.rb:48]: what do you mean?

**Comment by abpostelnicu** [comment]
@SMillerDev do you think you can approve this?

**Review by SMillerDev — DISMISSED** [review]
Thanks @abpostelnicu  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Comment by abpostelnicu** [comment]
@SMillerDev can you push this to the main branch?

**Comment by SMillerDev** [comment]
The bot will automatically do that (and I'm not currently near a PC)

**Comment by carlocab** [comment]
Unfortunately your PR cannot be auto-merged, since it doesn't follow our commit format. You needed to have two commits here; one for `llvm` and another for `include-what-you-use`.

I can merge this manually and split your commits for you, but I haven't worked out how to preserve authorship of split commits when doing that. If you're fine with me doing that, let me know. Otherwise you can split the commits yourself (but we'll need to wait for CI again).

**Comment by abpostelnicu** [comment]
@carlocab is this better? I did two commits, for each bottle.

**Comment by abpostelnicu** [comment]
I still see the flag `automerge-skip`.

**Review by carlocab — APPROVED** [review]
Yup, looks great. Thanks, @abpostelnicu.

**Comment by carlocab** [comment]
Btw, since you seem to know some things about the lldb build: would you happen to know how to enable the Python integration for lldb in the formula?

Using the `LLDB_ENABLE_PYTHON` flag just leads to explicit linkage with the build-time Python interpreter, which we don't want (this leads to segfaults at run time). See also #38526.

**Comment by abpostelnicu** [comment]
> Btw, since you seem to know some things about the lldb build: would you happen to know how to enable the Python integration for lldb in the formula?
> 
> Using the `LLDB_ENABLE_PYTHON` flag just leads to explicit linkage with the build-time Python interpreter, which we don't want (this leads to segfaults at run time). See also #38526.

I’ll answer your question with another question: why do we ship lldb when it’s so broken? What does lldb from brew has that the native lldb from Apple doesn’t have.
I think the right approach here is to unship lldb.

**Pull request closed without merging**